pyproject.toml

```diff
@@ -8,13 +8,13 @@ dynamic = ["version"]
 description = "Two-stage kNN imputation using synthetic data and treelists"
 readme = "README.md"
 license = ""
-requires-python = ">=3.8"
+requires-python = ">=3.9"
 authors = [{ name = "Matt Gregory", email = "matt.gregory@oregonstate.edu" }]
 dependencies = [
+    "ipykernel",
```

Comment:
I think I actually followed your lead on `naip-cnn` [here](https://github.com/lemma-osu/naip-cnn/blob/main/pyproject.toml#L29)!  But is that because notebooks are integral to `naip-cnn` whereas it isn't clear that they will be utilized in this project?  My intention is that this package will likewise have a number of notebooks to demonstrate the workflow (as you've done with `naip-cnn`), but I'm not very far down that road yet.  

But otherwise it should go here?
```
[tool.hatch.envs.default]
dependencies = ["pre-commit"]
```

Reply:
Touche! I've been pretty slapdash on that project because I'm not expecting it to ever be installed by "end users" - there are a bunch of dependencies (like pre-commit) that I really should have just put in the developer environment 😬 

For packages, my understanding of "best practice" is to stick to dependencies that are directly required for API functionality, with the understanding that users will probably be installing into an environment that already has ancillary packages they'll want.

> But otherwise it should go here?
   ```bash
   [tool.hatch.envs.default]
   dependencies = ["pre-commit"]
   ```

Yeah, that should make it accessible to anyone running through a hatch shell.

Reply:
Done!